book/_toc.yml

```diff
@@ -55,6 +55,7 @@
             - file: tutorials/lidar/index
               sections:
                 - file: tutorials/lidar/ICESat2_tutorial
+                - file: tutorials/lidar/ASO_data_tutorial
```

Comment:
@c-webster - Looks like your link in the table of contents does not match the file name of your tutorial. Hence it won’t show in the preview of this PR

Reply:
I've updated the notebook name and committed the change.